mptt/managers.py

```diff
@@ -672,6 +672,79 @@ def partial_rebuild(self, tree_id):
 
         self._rebuild_helper(pks[0], 1, tree_id)
 
+    @delegate_manager
+    def bulk_load(self, data, target=None, position='last-child'):
```

Comment:
From a cursory glance this method doesn't appear to support trees that use `order_insertion_by`, since it always appends nodes to the next available `tree_id`. Is that correct?

I imagine doing that would be much trickier. At a minimum I think this method should throw an error if you try to call it when using `order_insertion_by`